src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/NewAzureSqlManagedDatabase.cs

```diff
@@ -0,0 +1,184 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.ManagedDatabase.Model;
+using Microsoft.Azure.Commands.Sql.ManagedInstance.Model;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Rest.Azure;
+using System.Management.Automation;
+using System.Collections;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to create a new Azure Sql Managed Database
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmSqlManagedDatabase", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.Low)]
+    public class NewAzureSqlManagedDatabase : AzureSqlManagedDatabaseCmdletBase<AzureSqlManagedDatabaseModel>
+    {
+        protected const string CreateNewByNameAndResourceGroupParameterSet =
+            "Create a new Managed Database from cmdlet input parameters";
```

Comment:
parameter set names should not have spaces

Reply:
Changed